LM-5551 intermediate-aci/dne-dci-intermediate-aci-mission1_health-dashboard/dashboard/app/views.py

```diff
@@ -96,22 +96,21 @@ def fault_update():
     if request.method == 'POST':
         return jsonify(get_faults(request.form['app']))
 
-        
-@app.route('/verify_to_spark', methods=["GET"]) 
-def verify_to_spark(): 
-    message = "I completed the ACI Health Dashboard Mission Completed Successfully!" 
-    post_to_spark(message) 
+
+@app.route('/verify_to_webex_teams', methods=["GET"])
+def verify_to_webex_teams():
```

Comment:
I believe that above index.html should work, based on this additional change, does that sound right?